web/src/pages/map/doc.vue

```diff
@@ -61,6 +61,15 @@
         <p v-html="example"></p>
       </template>
 
+      <h6>👁️ <translate>Hide markers</translate></h6>
+      <p><translate-slot>
+        <span translate>Do not want to see {title} markers?</span>
+        <template v-slot:title>
+          <i>{{ title }}</i>
+        </template>
+      </translate-slot></p>
+      <button class="mb-3 btn btn-info btn-sm" v-on:click.stop.prevent="$emit('hide-item-markers', item)" translate>Hide from map</button>
```

Comment:
We does not use `translate` in this way. Please use ` <translate>foo</translate>`